doc/specs/GitHub-discussions-spec.md

```diff
@@ -0,0 +1,57 @@
+# GitHub discussions spec
+
+## What is GitHub Discussions
+
+GitHub Discussions is a feature that currently is in the closed beta status. It will be released until the end of the year. 
+
+It's a new tab in the GitHub repo page where people can start new threads like in a forum.
+
+The threads must be added to a category. The answer feature can be enabled or disabled for categorys. If enabled a person with triage permission or the creator of the thread can mark a reply as answer.
+
+You can sort threads in the overview for categories and for (un-)answered.
+
+One thread can be pinned on the main page of the discussion section. (One with rules or something like that).
+
+Threads uses the same numeration like PR and issues. So they can be linked by the others. A conversation can also be locked like in issues and PR.
+
+## Why do we need this for PowerToys
+
+1. Sometimes people just want to say thank you to the PT team (like in https://github.com/microsoft/PowerToys/issues/7873) and a thread in GitHub Discussions where every one can comment if he just want to say thank you, is much better for the users and the team than open every time a new issue for that.
+
+2. Questions: If people just have questions, it gets difficult by asking them in issues. It's more useful for both sides (team and user) when questions are asked in a different section. Because in GitHub discussion you can mark a reply as the answer and other users can find the answer quicker.
+
+3. Off-topic. Sometimes issue discussions get very and sometimes a little off-topic (like https://github.com/microsoft/PowerToys/issues/7916#issuecomment-724813783) And it would be better if you can open a thread and write "As seen in #XXXX I think the real reason for ... is ..."
```

Comment:
You are making me sad with that link: https://github.com/microsoft/PowerToys/issues/7916#issuecomment-724813783 😢

Reply:
It's just a example. I want to say that GH Discussions would be a better place for such comments. I don't want that they get removed.

Reply:
That's Ok